apparmor.d/profiles-s-z/udisksd

```diff
@@ -130,8 +130,7 @@ profile udisksd @{exec_path} flags=(attach_disconnected) {
   @{sys}/class/ r,
 
   @{sys}/devices/pci[0-9]*/**/{ata,usb,mmc}[0-9]/{,**/}uevent w,
-  @{sys}/devices/virtual/block/dm-[0-9]*/ w,
-  @{sys}/devices/virtual/block/dm-[0-9]*/** w,
+  @{sys}/devices/virtual/block/*/** r,
```

Comment:
Any logic behind this?
1. Please note this profile has the disks-write abstraction already.
2. Beware with changes in `udisksd` regarding disks, that the best way to break some system.

Reply:
With `mdadm` you can give your raid array any name you like, and udiskd has to write to it, so only` dm-[0-9]*` is not enough. I think the read permission without write is a typo, I'll fix that.